controller/src/main/java/io/pravega/controller/server/security/auth/UserPrincipal.java

```diff
@@ -7,7 +7,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.controller.server.rpc.auth;
+package io.pravega.controller.server.security.auth;
```

Comment:
same comment here - should we take the opportunity to move this to auth module as this is generic User Principal which is serializable.

Reply:
https://github.com/pravega/pravega/pull/5051/#discussion_r478259935